manifests/prepare/package.pp

```diff
@@ -33,6 +33,6 @@
     mode     => $mode,
     source   => $source,
     require  => File[$puppet_agent::params::local_packages_dir],
-    checksum => sha256lite,
+    checksum => sha256,
```

Comment:
do we need to set the checksum at all? Isn't sha256 the default?

Reply:
That's a good call, thanks Tim. I'll update the commit to remove the attribute entirely.